docs/wcs/relax.rst

```diff
@@ -418,3 +418,15 @@ The flag bits are:
   potentially unsafe and is not recommended at this time.
 
 - ``WCSHDO_SIP``: Write out Simple Imaging Polynomial (SIP) keywords.
+
+- ``WCSHDO_P12``, ``WCSHDO_P13``, ``WCSHDO_P14``, ``WCSHDO_P15``, ``WCSHDO_P16``, ``WCSHDO_P17``, ``WCSHDO_EFMT``
+
+  These constants control the precision of the WCS keywords returned by `to_header`.
```

Comment:
@nden - should be ``~astropy.wcs.WCS.to_header`` otherwise the sphinx build fails because of a warning